code/modules/overmap/exoplanets/decor/turfs/wood.dm

```diff
@@ -0,0 +1,13 @@
+/turf/simulated/floor/exoplanet/wood
```

Comment:
I am very much against duplicating all of these turf definitions to have /exoplanet/ and non-/exoplanet/ variants. It is pure ass and awful for like a million reasons.

If the problem is that (for example) konyang ruins fail the zas tests on airless planets, the solution should be to test konyang ruins only on konyang.

Reply:
Alternative would be to remove the /exoplanet/ subtype and just make all turfs act like current exoplanet turfs do (and copy the planet's atmos). But that is not good for other reasons as well.